bellatrix.core/src/main/java/solutions/bellatrix/core/utilities/PageObjectModel.java

```diff
@@ -0,0 +1,8 @@
+package solutions.bellatrix.core.utilities;
+
+
+public interface PageObjectModel<MapT, AssertsT> {
```

Comment:
this is a major change - we need to discuss it, or extract in a separate PR

Reply:
> this is a major change - we need to discuss it, or extract in a separate PR

All modules (android, ios, desktop, playwright, and web) have POM. It is a major building block of our framework.

From insights from a project which uses BELLATRIX, it seems convenient to search for a baseline by the name of a page object. 
![image](https://github.com/user-attachments/assets/17020fbc-40f1-4a1c-b24a-64c36984c0ab)
Here is the usage of the interface PageObjectModel; It couldn't be abstracted to simply Object, as it will interfere with other overload methods. It couldn't be abstracted to simply String, as it will interfere with other overload methods.

More information could be found at [VisualRegressionAssertions file](https://github.com/AutomateThePlanet/BELLATRIX-Java/pull/53/files#diff-43515dbf55821c66e481758f78292c63d360beb09a87bc4508306d921460ef1a)

The change is done due to the assumption that all Page Object Models inside projects would extend the provided "*Section" and "*Page" objects.
Another assumption which was made was that no major changes would occur to the "*Section" and "*Page" objects provided, but would rather be extended inside the project's module(s).